mmv1/products/vertexai/api.yaml

```diff
@@ -251,7 +251,7 @@ objects:
         description: Output only. The models deployed in this Endpoint. To add or remove DeployedModels use EndpointService.DeployModel and EndpointService.UndeployModel respectively. Models can also be deployed and undeployed using the [Cloud Console](https://console.cloud.google.com/vertex-ai/).
         output: true
         item_type: !ruby/object:Api::Type::NestedObject
-          name: deployedModels
+          name: deployedModel
```

Comment:
Oh weird, I wonder why these don't cause a diff. Maybe pluralization logic in MMv1?